### ISSUE TITLE
Course subtitles reviews for Simplified Chinese (zh-CN)

### ISSUE THREAD

**Issue by chenglu** [huggingface/course#390]
Hi there 👋

Let's translate the course to `zh-CN` so that the whole community can benefit from this resource 🌎!

Below are the subtitles that need review, the existing translation was translated by Google translate - let us know here if you'd like to translate any and we'll add your name to the list. Once you're finished, open a pull request and tag this issue by including `#issue-number` in the description, where `issue-number` is the number of this issue.

The recommended PR title is: `docs(zh-cn): Reviewed {{TITLE}}` 
like: `docs(zh-cn): Reviewed No.01 - Welcome to the Hugging Face course`

## Subtitles by Chapter

**0 - Setup & 1 - Transformer models**

- [ ] [No.01 - Welcome to the Hugging Face course](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/00_welcome-to-the-hugging-face-course.srt)
- [ ] [No.02 - The pipeline function](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/01_the-pipeline-function.srt)
- [ ] [No.03 - The carbon footprint of Transformers](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/02_the-carbon-footprint-of-transformers.srt)
- [ ] [No.04 - What is Transfer Learning?](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/03_what-is-transfer-learning.srt)
- [ ] [No.05 - The Transformer architecture](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/04_the-transformer-architecture.srt)
- [ ] [No.06 - Transformer models: Encoders](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/05_transformer-models-encoders.srt)
- [ ] [No.07 - Transformer models: Decoders](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/06_transformer-models-decoders.srt)
- [ ] [No.08 - Transformer models: Encoder-Decoders](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/07_transformer-models-encoder-decoders.srt)

**2 - Using 🤗 Transformers**

- [ ] [No.09 - What happens inside the pipeline function? (PyTorch)](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/08_what-happens-inside-the-pipeline-function-(pytorch).srt)
- [ ] [No.10 - What happens inside the pipeline function? (TensorFlow)](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/09_what-happens-inside-the-pipeline-function-(tensorflow).srt)
- [ ] [No.11 - Instantiate a Transformers model (PyTorch)](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/10_instantiate-a-transformers-model-(pytorch).srt)
- [ ] [No.12 - Instantiate a Transformers model (TensorFlow)](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/11_instantiate-a-transformers-model-(tensorflow).srt)
- [ ] [No.13 - Tokenizers overview](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/12_tokenizers-overview.srt)
- [ ] [No.14 - Word-based tokenizers](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/13_word-based-tokenizers.srt)
- [ ] [No.15 - Character-based tokenizers](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/14_character-based-tokenizers.srt)
- [ ] [No.16 - Subword-based tokenizers](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/15_subword-based-tokenizers.srt)
- [ ] [No.17 - The tokenization pipeline](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/16_the-tokenization-pipeline.srt)
- [ ] [No.18 - Batching inputs together (PyTorch)](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/17_batching-inputs-together-(pytorch).srt)
- [ ] [No.19 - Batching inputs together (TensorFlow)](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/18_batching-inputs-together-(tensorflow).srt)

**3 - Fine-tuning a pretrained model**

- [ ] [No.20 - Hugging Face Datasets overview (Pytorch)](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/19_hugging-face-datasets-overview-(pytorch).srt)
- [ ] [No.21 - Hugging Face Datasets overview (Tensorflow)](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/20_hugging-face-datasets-overview-(tensorflow).srt)
- [ ] [No.22 - Preprocessing sentence pairs (PyTorch)](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/21_preprocessing-sentence-pairs-(pytorch).srt)
- [ ] [No.23 - Preprocessing sentence pairs (TensorFlow)](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/22_preprocessing-sentence-pairs-(tensorflow).srt)
- [ ] [No.24 - What is dynamic padding?](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/23_what-is-dynamic-padding.srt)
- [ ] [No.25 - The Trainer API](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/24_the-trainer-api.srt)
- [ ] [No.26 - Keras introduction](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/25_keras-introduction.srt)
- [ ] [No.27 - Fine-tuning with TensorFlow](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/26_fine-tuning-with-tensorflow.srt)
- [ ] [No.28 - Learning rate scheduling with TensorFlow](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/27_learning-rate-scheduling-with-tensorflow.srt)
- [ ] [No.29 - TensorFlow Predictions and metrics](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/28_tensorflow-predictions-and-metrics.srt)
- [ ] [No.30 - Write your training loop in PyTorch](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/29_write-your-training-loop-in-pytorch.srt)
- [ ] [No.31 - Supercharge your PyTorch training loop with Accelerate](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/30_supercharge-your-pytorch-training-loop-with-accelerate.srt)

**4 - Sharing models and tokenizers**

- [ ] [No.32 - Navigating the Model Hub](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/31_navigating-the-model-hub.srt)
- [ ] [No.33 - Managing a repo on the Model Hub](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/32_managing-a-repo-on-the-model-hub.srt)
- [ ] [No.34 - The Push to Hub API (PyTorch)](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/33_the-push-to-hub-api-(pytorch).srt)
- [ ] [No.35 - The Push to Hub API (TensorFlow)](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/34_the-push-to-hub-api-(tensorflow).srt)

**5 - The 🤗 Datasets library**

- [ ] [No.36 - Loading a custom dataset](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/35_loading-a-custom-dataset.srt)
- [ ] [No.37 - Slice and dice a dataset 🔪](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/36_slice-and-dice-a-dataset-🔪.srt)
- [ ] [No.38 - Datasets + DataFrames = ❤️](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/37_datasets-+-dataframes-=-❤️.srt)
- [ ] [No.39 - Saving and reloading a dataset](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/38_saving-and-reloading-a-dataset.srt)
- [ ] [No.40 - Memory mapping & streaming](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/39_memory-mapping-&-streaming.srt)
- [ ] [No.41 - Uploading a dataset to the Hub](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/40_uploading-a-dataset-to-the-hub.srt)
- [ ] [No.42 - Text embeddings & semantic search](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/41_text-embeddings-&-semantic-search.srt)

**6 -  The 🤗 Tokenizers library**

- [ ] [No.43 - Training a new tokenizer](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/42_training-a-new-tokenizer.srt)
- [ ] [No.44 - Why are fast tokenizers called fast?](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/43_why-are-fast-tokenizers-called-fast.srt)
- [ ] [No.45 - Fast tokenizer superpowers](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/44_fast-tokenizer-superpowers.srt)
- [ ] [No.46 - Inside the Token classification pipeline (PyTorch)](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/45_inside-the-token-classification-pipeline-(pytorch).srt)
- [ ] [No.47 - Inside the Token classification pipeline (TensorFlow)](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/46_inside-the-token-classification-pipeline-(tensorflow).srt)
- [ ] [No.48 - Inside the Question answering pipeline (PyTorch)](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/47_inside-the-question-answering-pipeline-(pytorch).srt)
- [ ] [No.49 - Inside the Question answering pipeline (TensorFlow)](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/48_inside-the-question-answering-pipeline-(tensorflow).srt)
- [ ] [No.50 - What is normalization?](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/49_what-is-normalization.srt)
- [ ] [No.51 - What is pre-tokenization?](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/50_what-is-pre-tokenization.srt)
- [ ] [No.52 - Byte Pair Encoding Tokenization](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/51_byte-pair-encoding-tokenization.srt)
- [ ] [No.53 - WordPiece Tokenization](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/52_wordpiece-tokenization.srt)
- [ ] [No.54 - Unigram Tokenization](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/53_unigram-tokenization.srt)
- [ ] [No.55 - Building a new tokenizer](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/54_building-a-new-tokenizer.srt)

**7 - Main NLP tasks**

- [ ] [No.56 - Data processing for Token Classification](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/55_data-processing-for-token-classification.srt)
- [ ] [No.57 - Data processing for Masked Language Modeling](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/56_data-processing-for-masked-language-modeling.srt)
- [ ] [No.58 - What is perplexity?](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/57_what-is-perplexity.srt)
- [ ] [No.59 - What is domain adaptation?](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/58_what-is-domain-adaptation.srt)
- [ ] [No.60 - Data processing for Translation](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/59_data-processing-for-translation.srt)
- [ ] [No.61 - What is the BLEU metric?](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/60_what-is-the-bleu-metric.srt)
- [ ] [No.62 - Data processing for Summarization](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/61_data-processing-for-summarization.srt)
- [ ] [No.63 - What is the ROUGE metric?](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/62_what-is-the-rouge-metric.srt)
- [ ] [No.64 - Data processing for Causal Language Modeling](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/63_data-processing-for-causal-language-modeling.srt)
- [ ] [No.65 - Using a custom loss function](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/64_using-a-custom-loss-function.srt)
- [ ] [No.66 - Data processing for Question Answering](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/65_data-processing-for-question-answering.srt)
- [ ] [No.67 - The Post processing step in Question Answering (PyTorch)](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/66_the-post-processing-step-in-question-answering-(pytorch).srt)
- [ ] [No.68 - The Post processing step in Question Answering (TensorFlow)](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/67_the-post-processing-step-in-question-answering-(tensorflow).srt)
- [ ] [No.69 - Data Collators: A Tour](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/68_data-collators-a-tour.srt)

**8 - How to ask for help**

- [ ] [No.70 - What to do when you get an error?](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/69_what-to-do-when-you-get-an-error.srt)
- [ ] [No.71 - Using a debugger in a notebook](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/70_using-a-debugger-in-a-notebook.srt)
- [ ] [No.72 - Using a debugger in a terminal](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/71_using-a-debugger-in-a-terminal.srt)
- [ ] [No.73 - Asking for help on the forums](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/72_asking-for-help-on-the-forums.srt)
- [ ] [No.74 - Debugging the Training Pipeline (PyTorch)](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/73_debugging-the-training-pipeline-(pytorch).srt)
- [ ] [No.75 - Debugging the Training Pipeline (TensorFlow)](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/74_debugging-the-training-pipeline-(tensorflow).srt)
- [ ] [No.76 - Writing a good issue](https://github.com/huggingface/course/blob/main/subtitles/zh-CN/75_writing-a-good-issue.srt)

**Comment by xianbaoqian** [comment]
Great work. Thanks @chenglu !

**Comment by tyisme614** [comment]
Let's do it.

**Comment by jyizheng** [comment]
no 20 claimed

**Comment by iCell** [comment]
I'll take No.00 - No. 07 (8 in total).

**Comment by tyisme614** [comment]
chapter 7 is done.

**Comment by tyisme614** [comment]
take task No.31-No.34

**Comment by innovation64** [comment]
I'll take No.08

**Comment by yaoqih** [comment]
I'll take No.31

**Comment by tyisme614** [comment]
> I'll take No.31

sorry bro, No.31 is done.

**Comment by yaoqih** [comment]
> > I'll take No.31
> 
> sorry bro, No.31 is done. FYI, #451

I'm very sorry, I missed the comment information, it doesn't matter, I will look for the unselected ones

**Comment by PowerChina** [comment]
I'll take No.10

**Comment by chenglu** [comment]
The conflict is caused by the lack of the update to the main issue description since I just back to my "home office", @yaoqih could you please help out and review @tyisme614 's PR? (#451)

**Comment by yaoqih** [comment]
> The conflict is caused by the lack of the update to the main issue description since I just back to my "home office", @yaoqih could you please help out and review @tyisme614 's PR? (#451)

sure

**Comment by yaoqih** [comment]
I'll take No.69

**Comment by maybenotime** [comment]
I'll take No.24

**Comment by txyzy1232006** [comment]
No.25 claimed

**Comment by FYJNEVERFOLLOWS** [comment]
No. 29 Claimed

**Comment by FYJNEVERFOLLOWS** [comment]
No. 21 Claimed

**Comment by zhangchaosd** [comment]
No. 72 Claimed

**Comment by tyisme614** [comment]
chapter 5 has been finished. @chenglu

**Comment by tyisme614** [comment]
btw, chapter 4 has been finished long ago. @chenglu

**Comment by Vermillion-de** [comment]
Have talked offline with @chenglu , consider have a Spring Cleaning, so I will take care all the others left, thanks! 
Excited for the incoming release!
